src/rules/no-identical-functions.ts

```diff
@@ -0,0 +1,69 @@
+import { Rule } from "eslint";
+import * as estree from "estree";
+import { areEquivalent } from "../utils/equivalence";
+import { isBlockStatement } from "../utils/nodes";
+import { getMainFunctionTokenLocation } from "../utils/locations";
+
+const MESSAGE = "Update this function so that its implementation is not identical to the one on line {{line}}.";
+
+const rule: Rule.RuleModule = {
+  create(context: Rule.RuleContext) {
+    const functions: estree.Function[] = [];
+
+    return {
+      FunctionDeclaration(node: estree.Node) {
+        visitFunction(node as estree.FunctionDeclaration);
+      },
+      FunctionExpression(node: estree.Node) {
+        visitFunction(node as estree.FunctionExpression);
+      },
+      ArrowFunctionExpression(node: estree.Node) {
+        visitFunction(node as estree.ArrowFunctionExpression);
+      },
+
+      "Program:exit"() {
+        processFunctions();
+      },
+    };
+
+    function visitFunction(node: estree.Function) {
+      if (isBlockStatement(node.body) && isBigEnough(node.body)) {
```

Comment:
@stas-vilchik why did you keep `isBlockStatement`?

Reply:
to check that a function has at least one statement